### PR TITLE
UX update

### PR DESCRIPTION
**Pull request by niels9001**
This PR updates the icon and the overhaul look and feel of the app following the latest design guidance (and introduces a Light Theme :)).

### CODE REVIEW

File: samples/MvvmSampleUwp/Package.appxmanifest [hunk 1]
```diff
@@ -33,19 +33,18 @@
...
-        DisplayName="MVVM Toolkit Sample App"
-        Square150x150Logo="Assets\Square150x150Logo.png"
-        Square44x44Logo="Assets\Square44x44Logo.png"
-        Description="MVVM Toolkit Sample App"
-        BackgroundColor="#3750D1">
-        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png" Square71x71Logo="Assets\SmallTile.png" Square310x310Logo="Assets\LargeTile.png">
+        DisplayName="MVVM Toolkit Gallery"
+        Square150x150Logo="Assets\Icon\MedTile.png"
+        Square44x44Logo="Assets\Icon\AppList.png"
+        Description="MVVM Toolkit Gallery" BackgroundColor="transparent">
```
Comment (Sergio0694) [inline samples/MvvmSampleUwp/Package.appxmanifest:39]:
```suggestion
        Description="MVVM Toolkit Sample App" BackgroundColor="transparent">
```

File: samples/MvvmSampleUwp/Helpers/TitleBarHelper.cs [hunk 2]
```diff
@@ -28,11 +28,11 @@ public static void StyleTitleBar()
         titleBar.ButtonInactiveBackgroundColor = Colors.Transparent;
 
         // Theme aware colors
-        titleBar.ButtonForegroundColor = titleBar.ButtonHoverForegroundColor = titleBar.ButtonPressedForegroundColor = Colors.White;
-        titleBar.ButtonHoverBackgroundColor = Color.FromArgb(0x20, 0xFF, 0xFF, 0xFF);
-        titleBar.ButtonPressedBackgroundColor = Color.FromArgb(0x40, 0xFF, 0xFF, 0xFF);
-        titleBar.ButtonInactiveForegroundColor = Color.FromArgb(0xC0, 0xFF, 0xFF, 0xFF);
-        titleBar.InactiveForegroundColor = Color.FromArgb(0xA0, 0xA0, 0xA0, 0xA0);
+        //titleBar.ButtonForegroundColor = titleBar.ButtonHoverForegroundColor = titleBar.ButtonPressedForegroundColor = Colors.White;
```
Comment (Sergio0694) [inline samples/MvvmSampleUwp/Helpers/TitleBarHelper.cs:31]: Why are these commented out?
If no styling is needed anymore (why?), then these should just be removed.

**Review by Sergio0694 — CHANGES_REQUESTED** [review]

**Comment by Sergio0694** [comment]
The icon in the title bar isn't showing up for me, does its path need to be updated?

image

**Review by Sergio0694 — CHANGES_REQUESTED** [review]
This also needs updating:

https://github.com/CommunityToolkit/MVVM-Samples/blob/04d536a29894dab98dfc8f96efcaee3c7870f25e/samples/MvvmSampleUwp/Package.appxmanifest#L20

File: samples/MvvmSampleUwp/Package.appxmanifest (hunk 1 above)
Comment (niels9001) [reply samples/MvvmSampleUwp/Package.appxmanifest:39]: WinUI uses Gallery, that's why I tweaked it. We can revert that.

".. App" feels a bit redundant though.. "MVVM Toolkit Samples" ?

File: samples/MvvmSampleUwp/Helpers/TitleBarHelper.cs (hunk 2 above)
Comment (niels9001) [reply samples/MvvmSampleUwp/Helpers/TitleBarHelper.cs:31]: With light+dark theme we don't need those anymore. Removed!

File: samples/MvvmSampleUwp/Package.appxmanifest (hunk 1 above)
Comment (Sergio0694) [reply samples/MvvmSampleUwp/Package.appxmanifest:39]: This is for consistency with the Windows Community Toolkit Sample App, plus it's also the name I already used to reserve the package identity in the Store. I think it's fine for a first release, we can always update it later on when we coordinate with the Windows Community Toolkit as well 🙂

File: samples/MvvmSampleUwp/Shell.xaml [hunk 3]
```diff
@@ -173,12 +174,19 @@
...
-                    Height="16"
-                    Source="ms-appx:///Assets/StoreLogo.scale-400.png" />
-                <TextBlock VerticalAlignment="Center" Text="MVVM Toolkit Sample App" />
+                    Width="18"
+                    Margin="0,0,0,0"
+                    Source="ms-appx:///Assets/AppList.png" />
+                <TextBlock
+                    Margin="12,0,0,0"
+                    Style="{StaticResource CaptionTextBlockStyle}"
+                    Text="MVVM Toolkit Gallery" />
```
Comment (Sergio0694) [inline samples/MvvmSampleUwp/Shell.xaml:189]: This should also say "MVVM Toolkit Sample App" for now.

File: samples/MvvmSampleUwp/Package.appxmanifest (hunk 1 above)
Comment (niels9001) [reply samples/MvvmSampleUwp/Package.appxmanifest:39]: Fixed!

**Review by Sergio0694 — CHANGES_REQUESTED** [review]
Question: isn't this a tiny bit too small compared to eg. the one in the Store? 🤔
Also is it just me or is the text like, ever so slightly vertically misaligned?

image

**Review by Sergio0694 — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**